authentication/models.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+from django.contrib.auth.models import User
+# Create your models here.
+
+class UserProfile(models.Model):
+  uid = models.CharField(max_length=64, primary_key=True, editable=False)
+  user = models.OneToOneField(User, on_delete=models.CASCADE)
+  name=models.CharField(max_length=255)
+  email=models.EmailField(max_length=255)
```

Comment:
I saw the email is stored in the original `user` model. So, we can eliminate this field too?